ctru-rs/examples/hello-world.rs

```diff
@@ -29,9 +29,6 @@ fn main() {
         if hid.keys_down().contains(KeyPad::START) {
             break;
         }
-        // Flush and swap framebuffers
-        gfx.flush_buffers();
-        gfx.swap_buffers();
```

Comment:
you mentioned that the console handles flushing the screen automatically... does it do that whenever the stdout is flushed (e.g. on `\n` or explicit `flush()` calls)? 

I'd imagine that for most applications (non-console) the user would still want to flush + swap buffers every frame instead of letting underlying helpers do it at random times.

Edit: also, from looking at https://github.com/devkitPro/libctru/blob/master/libctru/source/console.c it seems the buffers are flushed by default but not normally swapped, which I guess doesn't matter since `consoleInit` also sets double-buffering off...

Reply:
We could specify that behaviour in a comment, since there is no need for manually flushing. Console does it on `print()`. I'll put this off for the docs PR.